src/App.tsx

```diff
@@ -1,25 +1,57 @@
-import React from 'react';
+// import React, { useState } from 'react';
+import { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
+// import { event } from 'cypress/types/jquery';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [lengthItem, setLengthItem] = useState('5');
+  const [currentPagin, setCurrentPagin] = useState(1);
+  const [startItem, setStartItem] = useState(1);
+
+  const lastPage = Math.ceil(items.length / +lengthItem);
+
+  const lastItemOfLastPage =
+    startItem + +lengthItem - 1 - (startItem + +lengthItem - 1 - items.length);
+
+  const lastItemOfPage = startItem + +lengthItem - 1;
+
+  const visibleItems = getNumbers(startItem, +lengthItem + startItem - 1).map(
+    n => `Item ${n}`,
+  );
+
+  const titleOfCount =
+    currentPagin === lastPage ? lastItemOfLastPage : lastItemOfPage;
+
+  const handlerItem = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
pls read [this](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) article and try to follow common naming convention in the future